packages/picasso/src/QuillEditor/formats/link.ts

```diff
@@ -0,0 +1,34 @@
+import { Classes } from '@toptal/picasso-shared'
+import Quill from 'quill'
+
+import getTypographyClassName from '../../Typography/utils/get-typography-class-name'
+
+const QuillLink = Quill.import('formats/link')
+
+class LinkBlot extends QuillLink {
+  static typographyClasses: Classes
+  static create(value: string) {
+    const node = super.create(value)
```

Comment:
Just for type safety
```suggestion
    const node: Element = super.create(value)
```

Reply:
Addressed :)